asciidoc-confluence-publisher-doc/etc/docs/00-index/10-plantuml.adoc

```diff
@@ -43,3 +43,32 @@ plantuml::../files/included-diagram.puml[]
 ....
 
 plantuml::../files/included-diagram.puml[]
+
+== C4-PlantUML
+
+https://github.com/plantuml-stdlib/C4-PlantUML[C4-PlantUML] diagrams are supported as well.
+
+----
+[plantuml, c4-diagram, svg]
```

Comment:
I realized that in my local test, the SVG is not displayed inline as image, but only linked as file. Could you please change the example from "svg" to "png" to make sure the image is properly displayed inline? I assume that this is the behaviour of the Confluence attachment macro, and might even be different in newer versions (I used a rather old version 6.0.5).

Reply:
Thank you for the adjustment, but please change it in both places (line 52 and 65).